### DISCUSSION TITLE
Is there a way to set tool config file for cross compile target test

### DISCUSSION THREAD

**Discussion by PegasusPlusUS** [nextest-rs/nextest#1885] [Q&A]
I was testing windows target compiled on Linux, 'cargo nextest run' currently try to directly run compiled target, however, usually that target should be consumed within virtual machine, docker, or wine. So is there way to config how to run cross compiled target to run test?

**Comment by sunshowers** [comment]
Yeah! Nextest supports target runners defined in the Cargo config: https://nexte.st/docs/features/target-runners/

But note however that this isn't truly first-class, for example there's no way to set per-test target runners. It would definitely be good to have that.
